src/main/java/frc/robot/utilities/DebugEntry.java

```diff
@@ -42,6 +43,8 @@ public DebugEntry(T defaultValue, String name, Subsystem subsystem) {
       localEntry = datalog.start("/" + subsystem.getName() + "/" + name, "boolean");
       localConsumer = (a) -> datalog.appendBoolean(localEntry, (Boolean) a, 0);
 
+    } else if (defaultValue instanceof ShooterMode) {
```

Comment:
We are passing in strings for shooterModeLog, not instances of ShooterMode.

Let's remove this workaround, and think about how to do this in a more elegant way